charts/kargo/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kargo
 description: Kargo
 type: application
-version: 0.1.0
-appVersion: v0.1.0
+version: 0.1.1
+appVersion: v0.6.0
```

Comment:
This should always be left alone. It is automatically set correctly by the release process.

Reply:
That may be, but if one tries to deploy eg. with ArgoCD from Git directly the missing appVersion causes the deployment to break. Thoughts?

Reply:
If you follow [this](https://kargo.akuity.io/contributor-guide/hacking-on-kargo#iterating-quickly) the chart should just work as is. It also makes it fast and seamless to rebuild and redeploy components as you hack away.

Reply:
I'll definitely check it out when I have the time, we also use Tilt, though your Tiltfile looks a lot more elaborate and I'll be sure to learn a thing or two! :)

That said, right now my workflow involves getting the deploy working via ArgoCD along with all the configs and manifests, so I must rely on pulling the chart from my fork for the time being.